scripts/functions.sh

```diff
@@ -8,8 +8,11 @@ function datef() {
 
 function createConfig() {
     cd "$APP_PERSIST_DIR"
-    CLIENT_ID="$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
+    ARG1=$1
+    DEFAULT_ID="$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)"
+    CLIENT_ID="${ARG1:=$DEFAULT_ID}"
```

Comment:
I more or less did what was mentioned [here](https://github.com/dockovpn/dockovpn/pull/163). I also modified how `start.sh` called this function.